helping-and-teaching.md

```diff
@@ -81,6 +77,5 @@ As a helper, you have a crucial role during workshops:
 ## See also
 
 * Carpentries instructor training
-* {doc}`Teaching Tech Together <teaching-tech-together>` chapters 8,
-  9, 10.
-  
\ No newline at end of file
+* [Teaching Tech Together](teaching-tech-together) chapters 8,
```

Comment:
@rkdarst, [Teaching Tech Together](teaching-tech-together) as well. Is this way of writing an internal link OK?